contracts/bridge/Bridge.sol

```diff
@@ -208,6 +265,27 @@ contract Bridge is IBridge, Ownable {
         return result;
     }
 
+    /// Verifies that the given data is a valid data on BandChain as of the relayed block height.
+    /// @param blockHeight The block height. Someone must already relay this block.
+    /// @param result The result of this request.
+    /// @param version Lastest block height that the data node was updated.
+    /// @param merklePaths Merkle proof that shows how the data leave is part of the oracle iAVL.
+    function verifyOracleData(
+        uint256 blockHeight,
+        Result calldata result,
```

Comment:
verifyOracleData receive result but verifyOracleResult receive data

Reply:
`verifyOracleResult` is an interface function from day one to be function to verify result of oracle from other contract. It will break other contracts.